uberenv.py

```diff
@@ -233,28 +240,42 @@ def __init__(self, opts, extra_opts):
 
         # load project settings
         self.project_opts = load_json_file(opts["project_json"])
-        print("[uberenv project settings: {}]".format(str(self.project_opts)))
-        print("[uberenv options: {}]".format(str(self.opts)))
+        print("[uberenv project settings: {0}]".format(str(self.project_opts)))
+        print("[uberenv options: {0}]".format(str(self.opts)))
+
+        # setup main package name
+        self.pkg_name = self.project_opts["package_name"]
 
     def setup_paths_and_dirs(self):
-        self.uberenv_path = os.path.dirname(os.path.realpath(__file__))
+        self.uberenv_path = os.path.split(os.path.abspath(__file__))[0]
 
-    def set_from_args_or_json(self,setting):
-        try:
-            setting_value = self.project_opts[setting]
-        except (KeyError):
-            print("ERROR: {} must at least be defined in project.json".format(setting))
-            raise
+        self.dest_dir = os.path.abspath(self.opts["prefix"])
+        print("[installing to: {0}]".format(self.dest_dir))
+
+        # print a warning if the dest path already exists
```

Comment:
This is not entirely safe... if dest_dir is a file it will mkdir in the place and fail.